### PR TITLE
CD: Publish APK to Github release

### PR DESCRIPTION
**Pull request by kayleegeorge**
https://github.com/daimo-eth/daimo/issues/1188

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -280,3 +280,20 @@ jobs:
...
+        run: eas build --platform android --profile preview --non-interactive
+        env:
+          EXPO_TOKEN: ${{ secrets.EXPO_TOKEN }}
+
+      - name: Download APK from EAS
+        run: |
+          eas artifact:download --platform android
+        working-directory: ./apps/daimo-mobile
+
+      - name: Publish APK to GitHub Releases
```
Comment (dcposch) [inline .github/workflows/cd.yml:294]: will this make a new GH release on every push to master?

**Review by dcposch — APPROVED** [review]
LGTM in general

recommend making a new GH release only for new app store releases. I'm not sure what the easiest way to accomplish this is--your call--might be possible to have it run when we push a tag eg `v1.10.0`

**Comment by kayleegeorge** [comment]
@dcposch changed to publish on new tags

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -280,3 +282,42 @@ jobs:
       - name: Build and publish update
         run: npm run build:prod
         working-directory: ./apps/daimo-mobile
+
+  # Only publish APK on tag (e.g. v1.0.0)
+  publish-apk:
+    if: startsWith(github.ref, 'refs/tags/v')
```
Comment (dcposch) [inline .github/workflows/cd.yml:288]: 🚀

**Review by dcposch — APPROVED** [review]

**Pull request merged**